sudo.cpp

```diff
@@ -161,6 +164,15 @@ int Sudo::main()
         {
             mBackend = BACK_SUDO;
             mArgs.removeAt(0);
+        } else if ("-r" == arg1)
+        {
+            mArgs.removeAt(0);
+            m_remove = "-r";
```

Comment:
Why comparing here, assigning the very same to `m_remove` (which I see as a very confusing name for local variable) and then the same comparison in `PasswordDialog` constructor?